quickstart/docker-compose.yaml

```diff
@@ -67,6 +67,8 @@ services:
         interval: 5s
         timeout: 20s
         retries: 5
+    ports:
+       - "7777:7777"
```

Comment:
[The original reason for not exporting ports was to avoid port conflicts with the host during startup.]

Could you please update https://github.com/linkedin/venice/blob/main/docs/quickstart/quickstart-single-datacenter.md#step-3-run-docker-compose  saying please change the port if there is a conflict?

Reply:
Should we add accessible ports to other components as well? For example, to the controller, for creating stores. If I want to run VPJ from outside of the Docker, then I suppose Kafka would also need to have an accessible port? It's ok to do it piecemeal if doing them all is too cumbersome. Just wondering.

Reply:
What do we need to do here? Just change the doc? Or is there more?

Reply:
bump

Reply:
Sorry, I missed this thread.

Actually I recently created the Venice Standalone runner, that is also an easy way to start Venice locally and also it is easier for local development.

I will try again this docker composed deployment and then I will update the docs.

About port conflicts
@sushantmane do you have other services on those ports ? or is it because you want to launch multiple instances of the router on separate isolated environments ?

Reply:
I think PulsarSink has been using 7777 in this commit? So, I guess we can either drop this PR or tweak it to a different number